internal/cri/server/container_create.go

```diff
@@ -167,6 +167,13 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 	if err != nil {
 		return nil, fmt.Errorf("failed to get sandbox runtime: %w", err)
 	}
+	var runtimeHandler *runtime.RuntimeHandler
+	for _, f := range c.runtimeHandlers {
+		f := f
+		if f.Name == sandbox.Metadata.RuntimeHandler {
+			runtimeHandler = f
```

Comment:
I think we can fast return here bacause `runtimeHandler` comes from map[string]Runtime.

Reply:
@fuweid that is added by the first two commits, part of #9787 . I based on that because it adds the plumbing needed, but it might be simpler if all the comments for those commits go to #9787? And here we just review the last two commits that I added.

What do you think?

Reply:
Sure. Let's merge #9787 first.